interactions/models/internal/extension.py

```diff
@@ -145,6 +151,14 @@ def listeners(self) -> List["Listener"]:
         """Get the listeners from this Extension."""
         return self._listeners
 
+    def load(self, *args, **kwargs):
```

Comment:
PEP 585 supports `[]` subscripting for generic aliasing returns, I propose modifying:

```suggestion
    def load(self, *args, **kwargs) -> Callable[[], ...]:
```